io/EptReader.hpp

```diff
@@ -101,16 +103,22 @@ class PDAL_DLL EptReader : public Reader, public Streamable
         PointId pointId) const;
 
     void readAddon(PointView& dst, const Key& key, const Addon& addon,
-        PointId startId) const;
+        PointId startId = 0) const;
 
     // To allow testing of hidden getRemoteType() and getCoercedType().
     static Dimension::Type getRemoteTypeTest(const NL::json& dimInfo);
     static Dimension::Type getCoercedTypeTest(const NL::json& dimInfo);
 
-    // For streamable pipeline.
+    // For streaming operation.
+    struct NodeBuffer;
+    using NodeBufferMap = std::map<uint64_t, std::unique_ptr<NodeBuffer>>;
```

Comment:
It's mostly something to serve as a unique identifier for the `key` since we insert empty `unique_ptr` instances as values into the map to indicate a currently loading node until it gets populated.

Reply:
I didn't understand why it wouldn't be a vector<unique_ptr<NodeBuffer>>

Reply:
Since we're erasing from random locations within the structure it doesn't fit the typical vector model.

Reply:
I guess then I'd use a list?  Using a key that you never use is confusing.

Reply:
Good call, switched to a `list` in 2f3dc3f.